build/cocos2d_libs.xcodeproj/project.pbxproj

```diff
@@ -17490,21 +17490,21 @@
 				HEADER_SEARCH_PATHS = "";
 				IPHONEOS_DEPLOYMENT_TARGET = 9.1;
 				LIBRARY_SEARCH_PATHS = (
-					"\"$(SRCROOT)/../external/png/prebuilt/ios\"",
```

Comment:
I merged the tvos libraries and created a [new release](https://github.com/cocos2d/cocos2d-x-3rd-party-libs-bin/releases/tag/v3-deps-131). You should update `external/config.json` to use new 3rd party library version: 

`"version": "v3-deps-131"`